packages/govuk-frontend/src/govuk/components/tag/_index.scss

```diff
@@ -16,11 +22,10 @@
     padding-right: 8px;
     padding-bottom: 3px;
     padding-left: 8px;
-
     color: govuk-shade(govuk-colour("blue"), 60%);
     background-color: govuk-tint(govuk-colour("blue"), 70%);
-
     text-decoration: none;
+    overflow-wrap: break-word;
```

Comment:
Can we add a Review app example to show long tags in action?

Bit like [Summary list extreme](https://govuk-frontend-review.herokuapp.com/components/summary-list/extreme/preview) 😆